src/controller/tests/logout.controller.test.ts

```diff
@@ -0,0 +1,25 @@
+import Logout from '../logout.controller'
+
+import Testing from '../../utils/testing.utils'
+
+// Testing Globals
+let req: any
+let res: any
+let next: any
+
+describe('Logout', () => {
+    beforeEach(() => {
+        req = Testing.fakeRequest()
+        res = Testing.fakeResponse()
+        next = Testing.fakeNext()
+    })
+
+    describe('GET - /logout', () => {
+        describe('200 - Ok', () => {
+            beforeEach(async () => {
+                await Logout(req,res,next) // testing logout
+            })
+            test('Status code is 200', () => expect(res.status).toHaveBeenCalledWith(200))
```

Comment:
let's add at least one more test here to make sure that `res.clearCookie` was called with 'refreshToken', that way if someone changes it we'll know.